hoplite-aws/src/main/kotlin/com/sksamuel/hoplite/aws/AwsSecretsManagerPreprocessor.kt

```diff
@@ -29,6 +29,7 @@ import kotlinx.serialization.json.Json
  */
 class AwsSecretsManagerPreprocessor(
   private val report: Boolean = false,
+  private val json: Json = Json.Default,
```

Comment:
I think if anyone bumps versions without recompiling this might cause a runtime error.
Should we add the original as an additional constructor and remove the default ?

Reply:
Thanks was not aware of that workaround!